CHANGELOG.md

```diff
@@ -15,6 +15,9 @@ Versioning](http://semver.org/spec/v2.0.0.html) except to the first release.
 
 ### Changed
 
+- Add `Call16` method, support build tag `go_tarantool_call_17`
```

Comment:
Please add a description of the tag to [README.md#build-tags](README.md#build-tags).


You can copy-paste changes in [README.md#build-tags](https://github.com/tarantool/go-tarantool/pull/174/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5) from #174 (but don't copy about go_tarantool_msgpack_v5) if you want. But I do not insist.